github/repos_pages.go

```diff
@@ -30,7 +30,7 @@ type PagesBuild struct {
 	Commit    *string     `json:"commit,omitempty"`
 	Duration  *int        `json:"duration,omitempty"`
 	CreatedAt *Timestamp  `json:"created_at,omitempty"`
-	UpdatedAt *Timestamp  `json:"created_at,omitempty"`
+	UpdatedAt *Timestamp  `json:"updated_at,omitempty"`
```

Comment:
Nice catch!

Do you mind factoring out this change into a separate PR? That way, it can be merged sooner. It's a trivial fix and not related to this PR.